osm2gtfs/creators/ci_abidjan/trips_creator_ci_abidjan.py

```diff
@@ -0,0 +1,209 @@
+
+        feed.SetDefaultServicePeriod(default_service_period)
+        # This sets the default agency to the agency in config.json
+        # Not ever used, but needed to avoid an error in feed.AddRoute() below
+        feed.GetDefaultAgency()
+
+        for route_ref, line in sorted(lines.iteritems()):
+            if not isinstance(line, Line):
+                continue
+            # print("Generating schedule for line: " + route_ref)
+
+            if 'operator' in line.tags and line.tags['operator']:
+                agency_id = line.tags['operator']
+                if agency_id == self.config['agency']['agency_name']:
+                    agency_id = self.config['agency']['agency_id']
+            else:
+                agency_id = 'Unknown Agency'
+
+            try:
+                agency = feed.GetAgency(agency_id)
+            except KeyError:
+                agency = self._init_agency(feed, agency_id)
+
+            line_gtfs = feed.AddRoute(
+                short_name=str(line.route_id),
+                long_name=line.name,
+                # we change the route_long_name with the 'from' and 'to' tags
+                # of the route as the route_master name tag contains
+                # the line code (route_short_name)
+                route_type=line.route_type,
+                route_id=line.osm_id)
+            line_gtfs.agency_id = agency.agency_id
+            line_gtfs.route_desc = ""
+            line_gtfs.route_color = "1779c2"
+            line_gtfs.route_text_color = "ffffff"
+
+            route_index = 0
+            itineraries = line.get_itineraries()
+            transport_hours = transporthours.main.Main()
+
+            line_hours_list = transport_hours.tagsToGtfs(line.tags)
+            line_hours_dict = self._process_transport_hours(feed, line_hours_list)
+
+            for a_route in itineraries:
+                itinerary_hours_list = transport_hours.tagsToGtfs(a_route.tags)
+
+                if itinerary_hours_list:
+                    itinerary_hours_dict = self._process_transport_hours(feed, itinerary_hours_list)
+                elif line_hours_dict:
+                    itinerary_hours_dict = line_hours_dict
+                else:
+                    itinerary_hours_dict = default_hours_dict
+
+                for service_id,itinerary_hours in itinerary_hours_dict.items():
+                    service_period = feed.GetServicePeriod(service_id)
+                    trip_gtfs = line_gtfs.AddTrip(feed, service_period=service_period)
+                    trip_gtfs.shape_id = self._add_shape_to_feed(feed, a_route.osm_id, a_route)
+                    trip_gtfs.direction_id = route_index % 2
+                    route_index += 1
+
+                    if a_route.fr and a_route.to:
+                        trip_gtfs.trip_headsign = a_route.to
+                        line_gtfs.route_long_name = a_route.fr + " ↔ ".decode('utf-8') +  a_route.to
+
+                    DEFAULT_TRAVEL_TIME = 120 # minutes
```

Comment:
you can put it on top of the class with the other constants

Reply:
ok